### ISSUE TITLE
Feature Request VPR or even all jobs? Alternative rotation settings?

### ISSUE THREAD

**Issue by Shiratama-works** [Nik-Potokar/XIVSlothCombo#1478]
Since many fights these days have off times and etc that shifts burst windows for later eg DSR P3and P6, P12S and etc. It would be cool to have a something to spam on that does not initiate the burst options but just simple 123 combos to charge up juices for a big burst or just keeping uptime.

For example currently VPR Sloth changes Steel Fangs to all in one rotation key, make Dread Fangs into a all in one conservekey or somekind?

**Comment by runawayturrtle** [comment]
Would love, love, love this for all jobs. Even just being able to put the combo'd rotation onto a (usually) useless button like leg sweep so that we have the option to conserve gauge and still use our base combos would be incredible. Regardless, phenomenal work, especially this soon after xpac launch. Thank you dev team!

**Comment by Crosis4028** [comment]
Yeah. I agree with the use case. It could be beneficial when your static decides to delay raid buffs/change raid buff timings when doing certain fights at the end-game.

Currently doing this manually by playing with ADVANCED mode but this would be easier to do if something like this was implemented - though I could see this being very job-specific in terms of implementation.

For Viper for instance. it would mean a separate button that just handles your normal combo sequence.

then when you feel like it, you can plop back to using the Single Target Simple Mode and let it "catch up".

While I am not a dev, I've played with the use case and logic behind it and I would assume that the SIMPLE TARGET SIMPLE MODE would very likely break a lot of things with the fact that it considers remaining time left before Serpent's Ire/Gauge when it comes to prioritizing which skills to use to time it properly for burst windows. that being said, the use case sounds very appealing. 

I'd be curious to know what Kage and gang think about it.

**Comment by Shiratama-works** [comment]
We actually we already have something simular for SMN and SGE not sure if it is what it is intend to. But it does stall the burst window untill u want

**Comment by Kagekazu** [comment]
as far as i know this will not happen, ever.

if u want to hold burst, as long as u dont spam the button it will always prioritize ur normal rotation for most jobs.

SMN is an exception to the rule, not sure if it will stay this way.
Its also being deleted from monk.

Healers are different, since they dont really have a burst rotation.

But it might be best to @Taurenkey  and let him have the final say in this.

**Comment by Akechi-kun** [comment]
Just going to say this now: 
### This plugin should **_NOT_** be used in Ultimates.

Sloth is not made or even tuned for Ultimate content as downtime is not calculated in any fight, in which each Ultimate fight has a minimum of 4 instances of uncalculated downtime. Furthermore, because downtime navigation is almost exclusively for this content, it's highly recommended against using in said content.

If you want to reduce the button bloat on your hotbars for high-end use, then sure... but if you want to spam one-button & turn off your brain for an Ultimate clear or good parse, you might want to look elsewhere for a different plugin.

**Comment by Shiratama-works** [comment]
> Just going to say this now:
> 
> ### This plugin should **_NOT_** be used in Ultimates.
> Sloth is not made or even tuned for Ultimate content as downtime is not calculated in any fight, in which each Ultimate fight has a minimum of 4 instances of uncalculated downtime. Furthermore, because downtime navigation is almost exclusively for this content, it's highly recommended against using in said content.
> 
> If you want to reduce the button bloat on your hotbars for high-end use, then sure... but if you want to spam one-button & turn off your brain for an Ultimate clear or good parse, you might want to look elsewhere for a different plugin.

My example might be missleading, Using it to get good parse is not my intention. it is just an idea as running dungeon also can get a use of this as it can delay CDs for a as the last pack before boss is nearly dead.

**Comment by Akechi-kun** [comment]
Sorry, I assumed Ultimates because you stated DSR as an example. 

Some jobs do support this kind of function, whereas a slider is added to stop using certain skills when target has reached a certain HP%. This is something that’s done as a case-by-case basis though. I work mainly on Tanks so I wouldn’t be the deciding factor there. However this would probably be the closest thing to what you’re asking for.
